docs/1.guide/1.index.md

```diff
@@ -310,6 +310,20 @@ await storage.getItem("k"); // => <string>
 
 storage.setItem("k", "val"); // Check ok
 storage.setItem("k", 123); // TS error
+
+type TestObjType = {
+  a: number;
+  b: boolean;
+};
+type MyStorage = {
```

Comment:
For future expansion of first generic, i prefer that we namespace it like `Storage<{ data: {} }>`

(also can you please add ts example to a new section in the end of page? 🙏🏼

Reply:
sorry, I don't understand, could you explain it more detail? I guess you mean to adjust the definition of getItem like:
```ts
type NsObject = {
  data: object
  [key: string]: any
}
export interface Storage<T extends StorageValue = StorageValue> {
  // ...
  getItem<U extends Extract<T, NsObject>, K extends keyof U["data"]>(
     key: K, ops?: TransactionOptions
  ): Promise<U["data"][K] | null>;
  getItem<U extends T>(key: string, opts?: TransactionOptions ): Promise<U | null>;
}
```

<img width="725" alt="image" src="https://github.com/user-attachments/assets/4da523d9-7020-4958-8150-999ac00cf191">

Reply:
I agree to use data as the namespace, I have updated.